data/json/overmap_specials.json

```diff
@@ -164,10 +164,10 @@
             { "point":[2,1,0], "overmap": "hive"},
             { "point":[2,2,0], "overmap": "hive"}
         ],
-        "locations" : [ "forest" ],
+        "locations" : [ "field", "forest" ],
```

Comment:
Nit, but I think you can just use `"wilderness"`, [which equates](https://github.com/CleverRaven/Cataclysm-DDA/blob/850b8a71a513d81fbde81fea2c194e437cf427be/src/overmap.cpp#L113) to `"field", "forest"`